src/registration/models/helper.py

```diff
@@ -354,8 +354,10 @@ def all_jobs(self):
         return jobs
 
     @property
-    def ask_nutrition(self) -> bool:
-        """Returns True if the nutritional prefernce is required."""
+    def ask_nutrition(self):
+        """Returns True if the nutritional preference is required."""
+        if hasattr(self, "event") and not self.event.ask_nutrition:
```

Comment:
When would `self.event` not be set? Seems a bit suboptimal to rely on `hasattr()` here.